destination/destination_test.go

```diff
@@ -93,29 +94,49 @@ func TestDestination_Write(t *testing.T) {
 		ctrl := gomock.NewController(t)
 		ctx := context.Background()
 
-		record := sdk.Record{
+		metadata := sdk.Metadata{}
+		metadata.SetCreatedAt(time.Now())
+
+		records := make([]sdk.Record, 2)
+		records[0] = sdk.Record{
 			Position: sdk.Position("1.0"),
```

Comment:
sdk.Position("1.0") please change to Oracle position or set nil. It is not needed for destination

Reply:
Deleted.